src/user/user.service.ts

```diff
@@ -19,8 +19,39 @@ export class UserService {
     private readonly configService: ConfigService
   ) { }
 
+  async findUserById(id: number): Promise<User | undefined> {
+    if (id <= 0) {
```

Comment:
This is okay, but [there is a decorator](https://github.com/typestack/class-validator?tab=readme-ov-file#validation-decorators:~:text=divisible%20by%20another.-,%40IsPositive(),-Checks%20if%20the) to check if a value a positive number in your DTO